webapp/client/src/components/SubHeading.js

```diff
@@ -0,0 +1,11 @@
+import styled from "styled-components";
```

Comment:
Is there a Storybook story that shows this component? If not, please add one, maybe creating a new category heading called "Elements" (because this isn't specific to forms).